lib/utils/docker.coffee

```diff
@@ -110,13 +110,34 @@ exports.tarDirectory = tarDirectory = (dir) ->
 		pack.finalize()
 		return pack
 
+cacheHighlightStream = ->
+	colors = require('colors')
+	es = require('event-stream')
+	{ EOL } = require('os')
+
+	extractArrowMessage = (message) ->
+		arrowTest = /^\s*-+>\s*(.+)/i
+		if (match = arrowTest.exec(message))
+			match[1]
+		else
+			false
+
+	es.map (data, cb) ->
+		msg = extractArrowMessage(data)
+		if msg and msg.toLowerCase() == 'using cache'
+			data = colors.bgGreen.black(msg)
```

Comment:
Playing around with `colors`, it looks like if `msg` already has a color this won't work. Are we sure it always doesn't?